### PR TITLE
2way sync: sync only on wifi

### PR DESCRIPTION
**Pull request by tobiasKaminsky**
- [ ] Tests written, or not not needed

### CODE REVIEW

File: app/src/main/java/com/nextcloud/client/jobs/InternalTwoWaySyncWork.kt [hunk 1]
```diff
@@ -35,7 +35,9 @@ class InternalTwoWaySyncWork(
         var result = true
 
         if (powerManagementService.isPowerSavingEnabled ||
-            !connectivityService.isConnected || connectivityService.isInternetWalled
+            !connectivityService.isConnected ||
+            connectivityService.isInternetWalled ||
+            !connectivityService.connectivity.isWifi
```
Comment (bobemoe) [inline app/src/main/java/com/nextcloud/client/jobs/InternalTwoWaySyncWork.kt:40]: Is this going to prevent sync on Ethernet?

Comment (rugk) [reply app/src/main/java/com/nextcloud/client/jobs/InternalTwoWaySyncWork.kt:40]: Yeah should maybe check for metered connections, better? (Also for some wifi networks you don't want it to sync)

Comment (rugk) [reply app/src/main/java/com/nextcloud/client/jobs/InternalTwoWaySyncWork.kt:40]: Though maybe open a new issue for that to not get it lost? or can @tobiasKaminsky et al bring some insight in here?

**Pull request merged**